README.md

```diff
@@ -16,15 +16,16 @@ npm run build && open index.html
 More ticks doesn't mean "better", it depends on your needs.
 For example, if a package supports the css file extraction you can run the autoprefixing at build time.
 
-| Package | Version | Automatic Vendor Prefixing | Pseudo Classes | Media Queries | Styles As Object Literals | Extract CSS File |
-|---------|:-------:|:--------------------------:|:--------------:|:-------------:|:-------------------------:|:----------------:|
+| Package | Version | Automatic Vendor Prefixing | Pseudo Classes | Media Queries | Styles As Object Literals | Extract CSS File | Emit CSS only when component rendered for the first time | CSS property shorthand notation |
```

Comment:
No need to add new columns for a single package, please remove them.

Reply:
removed